.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug Report
+description: Create a report to help us reproduce and fix the bug
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Before submitting a bug, please make sure the issue hasn't been already addressed by searching through [the
+        existing and past issues](https://github.com/facebookresearch/torchsnapshot/issues?q=is%3Aissue+sort%3Acreated-desc+).
+  - type: textarea
+    attributes:
+      label: 🐛 Describe the bug
+      description: |
+        Please provide a clear and concise description of what the bug is.
+
+        If relevant, add a minimal example so that we can reproduce the error by running the code. It is very important for the snippet to be as succinct (minimal) as possible, so please take time to trim down any irrelevant code to help us debug efficiently. We are going to copy-paste your code and we expect to get the same result as you did: avoid any external data, and include the relevant imports, etc. For example:
+
+        ```python
+        # All necessary imports at the beginning
+        import torch
+        import torchsnapshot
+
+        # A succinct reproducing example trimmed down to the essential parts
+
+        ```
+
+        If the code is too long (hopefully, it isn't), feel free to put it in a public gist and link it in the issue: https://gist.github.com.
+
+        Please also paste or describe the results you observe instead of the expected results. If you observe an error, please paste the error message including the **full** traceback of the exception. It may be relevant to wrap error messages in ```` ```triple quotes blocks``` ````.
+      placeholder: |
+        A clear and concise description of what the bug is.
+
+        ```python
+        Sample code to reproduce the problem
+        ```
+
+        ```
+        The error message you got, with the full traceback.
+        ```
+  #   validations:
+  #     required: true
+  # - type: textarea
+  #   attributes:
+  #     label: Versions
+  #     description: |
+  #       Please run the following and paste the output below. Make sure the version numbers of all relevant packages (e.g. torch, torchdata, other domain packages) are included.
+  #       ```sh
+  #       wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
+  #       # For security purposes, please check the contents of collect_env.py before running it.
+  #       python collect_env.py
+  #       ```
+  #   validations:
+  #     required: true
```

Comment:
it will be useful to know the torchsnapshot + pytorch versions used as part of the bug report

Reply:
Resolved: added option to get version info.